examples/README.md

```diff
@@ -0,0 +1,141 @@
+# How to use the library
+
+This library provide the tools to use together a Redpanda schema registry and a Rabbitmq queue system to 
+consume and publish messages.
+
+# Consumers
+
+A consumer is a process that has the functionality of removing a message from the top of the queue,
+ process it, and go to the next message, doing the same with any subsequent messages until the queue
+is empty. In that moment it stays waiting until new messages are published in the queue, so it can start 
+again.
+
+## How a LabShare consumer process works
+
+A LabShare consumer adds some extra process to the default consumer behaviour. The process of a message using a LabShare consumer can be structured in different stages:
+
+* Message decoding: in this stage the consumer will read the headers attached to the message, and will 
+use them to deserialize the contents of the message, by using the format and compresion codec specified 
+in the headers.
+
+* Schema validation: after the contents of the message has been extracted, it is validated using the 
+schemas provided by the Redpanda Schema registry. To do that, it reads from the headers ... the subject name
+and version number of the schema the message was encoded into. With these two fields it will request
+the Redpanda service for the schema published using that subject and version number. The schema downloaded
+is then processed by the Fastavro library to validate the correction of the message contents.
+
+* Message processing: if the validation is correct, a new instance of the Processor class defined in 
+the PROCESSORS field of the config is created and the library will call the method process() with the
+contents of the message. Any required behaviour for the processing of the message can be added inside this
+method. If the process is correct (returns True), the message is purged, but if the process is incorrect (returns False) the message is sent to the dead letters queue.
+
+## Setting up a consumer
+
+To set up a consumer first we need to define the configuration settings that the consumer will use to run.
+These are the settings required for a consumer:
+
+You can copy file ```examples/example1/config.py``` and modify with your settings.
```

Comment:
```suggestion
You can copy file `examples/example1/config.py` and modify with your settings.
```

Reply:
I'll use references to the actual file so is easier to navigate

Reply:
I mean, that I'll change it to use Markdown links, like [...|...]

Reply:
I think links in Markdown are `[test for the link](https://url.com)`

Reply:
Thanks :)